tests/integration/test_keeper_path_acl/test.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from helpers.cluster import ClickHouseCluster
+from helpers.keeper_utils import get_fake_zk
+from kazoo.security import Id, ACL
+
+
+cluster = ClickHouseCluster(__file__)
+
+node = cluster.add_instance(
+    "node",
+    main_configs=["configs/keeper_config.xml", "configs/zookeeper_config.xml"],
+    stay_alive=True,
+)
+
+
+@pytest.fixture(scope="module", autouse=True)
+def started_cluster():
+    try:
+        cluster.start()
+        yield cluster
+    finally:
+        cluster.shutdown()
+
+
+def test_path_acl(started_cluster):
+    """Test path_acl configuration parsing and ACL application"""
+    node.query(
+        "CREATE TABLE IF NOT EXISTS test_table (id UInt64) ENGINE = ReplicatedMergeTree('/test_path', 'replica1') ORDER BY id"
```

Comment:
Just to make sure we always create a table and don't use the same table in repeated tests. Yes, it shouldn't be a problem, but I guess in each run we want to test a newly created node.
```suggestion
        """
        DROP TABLE IF EXISTS test_table SYNC;
        CREATE TABLE IF NOT EXISTS test_table (id UInt64) ENGINE = ReplicatedMergeTree('/test_path', 'replica1') ORDER BY id;
        """
```